lib/bpf/ubpf_hashmap.c

```diff
@@ -77,7 +77,7 @@ ubpf_hashmap_dump(const struct ubpf_map *map, void **data)
     int value_size = map->value_size;
     int key_rounded_size = round_up(map->key_size, 8);
 
-    void *tmp_data = *data;
+    void *tmp_data = data;
```

Comment:
I think you can use `data` directly here, no?

Reply:
Yes, after last changes this pointer is unnecessary.